packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -1,4 +1,7 @@
 .jurisdiction-modal {
+    &__header ~ .dc-mobile-dialog__content {
+        height: 100%;
+    }
```

Comment:
If you are overwriting the dc class for safety you can do this. It means you are only tweaking this class which has parent `jurisdiction-modal__header`
```suggestion
    &__header {
    .dc-mobile-dialog__content {
        height: 100%;
        }
    }
```

Reply:
![image](https://github.com/binary-com/deriv-app/assets/108507236/977b4cde-ffba-4449-b4fe-ab48e26c38d7)

in order to implement the suggestion it has to be 

```
.dc-mobile__container {
   .dc-mobile__content {
      height:100%;
  }
}
```

which is not possible because `.dc-mobile__container` class comes from the modal component itself and will override other modals, this is why I added a custom `header_classname = jurisdiction-modal__header` and grab its sibling content to position

Reply:
![image](https://github.com/binary-com/deriv-app/assets/108507236/977b4cde-ffba-4449-b4fe-ab48e26c38d7)

in order to implement the suggestion, `.dc-mobile-dialog__content` has to. be a child of `__header` but based on how it is rendered, they're on sibling level hence ~ is used